net/dnsfallback/dnsfallback.go

```diff
@@ -30,7 +30,11 @@ func Lookup(ctx context.Context, host string) ([]netaddr.IP, error) {
 		ip      netaddr.IP
 	}
 
-	dm := derpmap.Prod()
+	dm, err := tailscale.ProdDERPMap(ctx, &http.Client{})
+	if err != nil {
+		// TODO I still think it's nice to be able to be able to fallback on a static thing here.
```

Comment:
Yeah, crap, I forgot about the DNS fallback mechanism.

Depending on DNS working breaks the whole purpose of this mechanism to get people up and running when their DNS is broken.

Hmm, let me think on this. Maybe we need to keep a cache of the last known derp map in /var/lib/tailscale as a json file that we can use instead of hitting tailscaled (this code is used by tailscaled when starting, so using the `client/tailscale` package within tailscaled doesn't really work)